Makefile

```diff
@@ -12,6 +12,12 @@ include $(config)
 export LDFLAGS = -pthread -lm
 export CFLAGS = -std=c++11 -Wall -O2 -Iinclude -fPIC
 
+ifdef DMLC_CORE_PATH
```

Comment:
allow dmlc_core path to be NONE, to be consistent with rest of config, see ADD_CFLAGS